webapp/platform/client/src/client4.ts

```diff
@@ -2345,13 +2345,14 @@ export default class Client4 {
 
     // General Routes
 
-    ping = () => {
+    ping = (getServerStatus: boolean, deviceId?: string) => {
```

Comment:
Do we need to add the device ID here? I don't think that's used on web. 0/5 if it's just for consistency though

Reply:
1/5 that I like the consistency with the server code.

Reply:
That makes sense to me. I forgot this was also in the client package which we want other people to use